examples/real-world/containers/RepoPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { loadRepo, loadStargazers } from '../actions';
+import Repo from '../components/Repo';
+import User from '../components/User';
+import List from '../components/List';
+
+function loadData(props) {
+  const { fullName } = props;
+  props.loadRepo(fullName, ['description']);
+  props.loadStargazers(fullName);
+}
+
+class RepoPage extends Component {
+  constructor(props) {
+    super(props);
+    this.renderUser = this.renderUser.bind(this);
+    this.handleLoadMoreClick = this.handleLoadMoreClick.bind(this);
+  }
+
+  componentWillMount() {
+    loadData(this.props);
```

Comment:
@gaearon Looking at this example made me think about how to prevent duplicate fetching of data when you have multiple views all interested in the same information (triggering actions in `componentWillMount`, like you do here), so I made this: [redux-requests](https://github.com/idolize/redux-requests). Would love to get your feedback on it!